.github/workflows/main_ci.yml

```diff
@@ -4,9 +4,11 @@ on:
   push:
     branches:
       - pydarshan-devel
+      - pydarshan-MiB
   pull_request:
     branches:
       - pydarshan-devel
+      - pydarshan-MiB
```

Comment:
this isn't helpful--there is no pull request against `pydarshan-MiB` here--maybe check the GitHub actions docs for how to enable CI against any branch, probably an asterisk or similar..